cockatrice/src/game/board/card_drag_item.cpp

```diff
@@ -47,8 +47,19 @@ void CardDragItem::updatePosition(const QPointF &cursorScenePos)
         cursorZone = zoneViewZone;
     else if (cardZone)
         cursorZone = cardZone;
-    if (!cursorZone)
+    if (!cursorZone) {
+        // Temporary fix: avoid the cards getting stuck visually when not over
```

Comment:
what would the permanent fix be?

Reply:
I'm rewriting the whole drag-and-drop system for #4974 so that's what I had in mind. The "temporary" doesn't really make sense in the context of the code, I'll drop it.